### DISCUSSION TITLE
Any plans for a Cyberduck iOS/iPadOS app?

### DISCUSSION THREAD

**Discussion by r0hin** [iterate-ch/cyberduck#14940] [Ideas]
By far the best remote file system client for MacOS, but many users may want to access their servers or cloud storage on-the-go.

Are there any plans to port the existing Cyberduck for MacOS build to iOS/iPadOS as well?

Perhaps something like Mac Catalyst is available, such that the codebase remains shared and relatively little development work is required.

**Comment by dkocher** [comment]
The best option would probably be to have a FileProvider API implementation [^1] to allow access form any other iOS application including the Files app [^2].

[^1]: https://developer.apple.com/documentation/fileprovider/
[^2]: https://support.apple.com/en-us/HT206481

**Comment by johnlabor** [comment]
Hi @dkocher it has been a while since this discussion thread. Any plans?

Right now it is really hard to use raw cryptomator with other third party s3 mount on ios with a lot of limitations.
This would be an ideal solution to that. Thanks.
